docs/source/tutorials/fashion_mnist.ipynb

```diff
@@ -0,0 +1,1274 @@
+    "#     !pip install \"cleanlab[datalab] @ git+https://github.com/cleanlab/cleanlab.git\"\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Package installation (this cell is hidden from docs.cleanlab.ai).\n",
+    "# If running on Colab, may want to use GPU (select: Runtime > Change runtime type > Hardware accelerator > GPU)\n",
+    "\n",
+    "dependencies = [\"cleanlab\", \"matplotlib\", \"torch\", \"torchvision\", \"datasets\"]\n",
+    "\n",
+    "if \"google.colab\" in str(get_ipython()):  # Check if it's running in Google Colab\n",
+    "    %pip install cleanlab  # for colab\n",
+    "    cmd = ' '.join([dep for dep in dependencies if dep != \"cleanlab\"])\n",
+    "    %pip install $cmd\n",
+    "else:\n",
+    "    missing_dependencies = []\n",
+    "    for dependency in dependencies:\n",
+    "        try:\n",
+    "            __import__(dependency)\n",
+    "        except ImportError:\n",
+    "            missing_dependencies.append(dependency)\n",
+    "\n",
+    "    if len(missing_dependencies) > 0:\n",
+    "        print(\"Missing required dependencies:\")\n",
+    "        print(*missing_dependencies, sep=\", \")\n",
+    "        print(\"\\nPlease install them before running the rest of this notebook.\")\n",
+    "\n",
+    "# Suppress benign warnings: \n",
+    "import warnings \n",
+    "warnings.filterwarnings(\"ignore\", \"Lazy modules are a new feature.*\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from torch.utils.data import DataLoader\n",
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torch.nn.functional as F\n",
+    "import torch.optim as optim\n",
+    "\n",
+    "from sklearn.model_selection import KFold\n",
+    "import numpy as np\n",
+    "\n",
+    "from tqdm.auto import tqdm\n",
+    "\n",
+    "from cleanlab import Datalab\n",
+    "from datasets import load_dataset, concatenate_datasets\n",
+    "import matplotlib.pyplot as plt\n",
+    "import math"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# This (optional) cell is hidden from docs.cleanlab.ai\n",
+    "\n",
+    "SEED = 123  # for reproducibility\n",
+    "np.random.seed(SEED)\n",
+    "torch.manual_seed(SEED)\n",
+    "torch.backends.cudnn.deterministic = True\n",
+    "torch.backends.cudnn.benchmark = False\n",
+    "torch.cuda.manual_seed_all(SEED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set device\n",
+    "device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
+    "\n",
+    "\n",
+    "# CNN net\n",
+    "class Net(nn.Module):\n",
```

Comment:
why is the neural net being defined multiple times in the notebook?

Reply:
I see the second neural network code block lower down is in a nice dropdown, but not sure why there is this entirely other neural net code block earlier on

Reply:
That was by mistake, missed it in moving blocks around.